docs/content/development/extensions.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: doc_page
+---
+
+# Druid extensions
+
+Druid implements an extension system that allows for adding functionality at runtime. Extensions
+are commonly used to add support for deep storages (like HDFS and S3), metadata stores (like MySQL
+and PostgreSQL), new aggregators, new input formats, and so on.
+
+Production clusters will generally use at least two extensions; one for deep storage and one for a
+metadata store. Many clusters will also use additional extensions.
+
+## Including extensions
+
+Core extensions are bundled with the Druid tarball.
+
+Community extensions be download locally via the [pull-deps](../operations/pull-deps.html) tool. 
+Make sure to download the dependencies from a tag or branch that matches the version of Druid you have deployed.
+
+To enable an extensions, include the extension name in `druid.extensions.loadList`.
+
+For more information, please see [here](../operations/including-extensions.html). 
+
+## Core extensions
+
+Core extensions are maintained by Druid committers.
+
+|Name|Description|Docs|
+|----|-----------|----|
+|druid-avro-extensions|Support for data in Apache Avro data format.|[link](../ingestion/index.html)|
+|druid-datasketches|Support for approximate counts and set operations with [DataSketches](http://datasketches.github.io/).|[link](../development/datasketches-aggregators.html)|
+|druid-hdfs-storage|Support for data in Apache Avro data format.|[link](../ingestion/index.html)|
+|druid-histogram|HDFS deep storage.|[link](../dependencies/deep-storage.html#hdfs)|
+|druid-kafka-eight|Kafka ingest firehose (high level consumer).|[link](../ingestion/firehose.html#kafkaeightfirehose)|
+|druid-kafka-extraction-namespace|Kafka namespaced lookup.|[link](../querying/lookups.html#kafka-namespaced-lookup)|
+|druid-namespace-lookup|Namespaced lookups.|[link](../querying/lookups.html)|
+|druid-s3-extensions|S3 deep storage.|[link](../dependencies/deep-storage.html#s3-compatible/)|
+|mysql-metadata-storage|MySQL metadata store.|[link](../dependencies/metadata-storage.html#setting-up-mysql)|
+|postgresql-metadata-storage|PostgreSQL metadata store.|[link](../dependencies/metadata-storage.html#setting-up-postgresql)|
+
+# Community Extensions
```

Comment:
also it looks like some of these groupIds are `io.druid.extensions.contrib` and some are `io.druid.extensions`.